mlibrary/catalog-browse
# Libsearch 748 hardening

# Overview

This pull request is part of [LIBSEARCH-748](https://mlit.atlassian.net/browse/LIBSEARCH-748).

It adds rack and puma prometheus exports
It updates npm in Dockerfiles
It adds a liveness url
Fixes display of material symbols for Firefox on Linux

## Testing

- Run the tests to make sure they pass 
(`docker-compose run --rm web bundle exec rspec`).
- Check the export url: localhost:9394/metrics
- Check the liveness url: localhost:4567/-/live

## Other notes
@erinesullivan I definitely want you to look over the change to the Material Symbols font.
